scripts/bash_includes/tc/lua

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: BSD-2-Clause
+
+function download_lua {
```

Comment:
I'm happy to see that, overall, your patch is aligned with Tilck's code style :+1: 
Of course, there are some details to fix.

1. While there's nothing really wrong with this function, I feel like an overkill to have params. Just hard-code the version in a local variable. Also, no need for a lua="lua" variable.

2. While there's nothing wrong with the "${varname}" syntax, it's just too heavy and unnecessary most of the time. In Tilck, I use naked variables for plain things, like version, tarname etc., while I use the "$var" syntax for paths and URLs.

For example:
```
   download_file_in_cache "$url" $tarname lua
   extract_cachefile_tar_gz $tarname lua-${version} lua
```

Reply:
Hey, I have updated the function as per the comment.